backend/core/src/listings/combined/combined-listing-filter-params.ts

```diff
@@ -24,5 +24,25 @@ export class CombinedListingFilterParams extends ListingFilterParams {
         return undefined
     }
   })
-  [CombinedListingFilterKeys.isExternal]?: boolean
+  [CombinedListingFilterKeys.isExternal]?: boolean;
+
+  @Expose()
+  @ApiProperty({
+    type: String,
+    example: "Santa Clara",
+    required: false,
+  })
+  @IsOptional({ groups: [ValidationsGroupsEnum.default] })
+  @IsString({ groups: [ValidationsGroupsEnum.default] })
+  [CombinedListingFilterKeys.county]?: string;
+
+  @Expose()
+  @ApiProperty({
+    type: String,
+    example: "San Jose",
+    required: false,
+  })
+  @IsOptional({ groups: [ValidationsGroupsEnum.default] })
+  @IsString({ groups: [ValidationsGroupsEnum.default] })
+  [CombinedListingFilterKeys.city]?: string
```

Comment:
we can keep this here although it is not a search filter we will support fyi

Reply:
Ah, okay.  That was listed as a P1 in the data sync doc, which is why I added it here